client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -103,6 +104,19 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 			} );
 		}
 
+		const cartItemForStorageAddOn = cartItems?.find(
+			( items ) => items.product_slug === PRODUCT_1GB_SPACE
+		);
+
+		if ( cartItemForStorageAddOn ) {
+			setProductCartItems( [ cartItemForStorageAddOn ] );
+
+			cartItemForStorageAddOn?.extra &&
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/83771/commits/9469b865704985f19993d75409c2efd61ae20a90

After some review, and testing to verify, the tracks event invoked by this condition is actually redundant. Because of this I remove the logic entirely